docs/deployments/predictors.md

```diff
@@ -362,6 +362,54 @@ The pre-installed system packages are listed in [images/onnx-predictor-cpu/Docke
 
 If your application requires additional dependencies, you can install additional [Python packages](python-packages.md) and [system packages](system-packages.md).
 
+## API requests
+
+The `payload` parameter in the `predict(self, payload)` method can be of different types depending on the content type of the request.
+
+Here are some examples:
+
+```bash
+# payload parameter is a dictionary object
```

Comment:
I think you should describe these examples in the form: if you make this kind of request, the payload type will be this e.g. 
The payload parameter type will be a python object if a request with a JSON payload is made:

The payload parameter type will be a bytes object if a request with "Content-Type: application/octet-stream" is made

etc.